paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -403,7 +404,7 @@ def print_rollback_cmd(
         )
 
 
-def paasta_mark_for_deployment(args: argparse.Namespace) -> None:
+def paasta_mark_for_deployment(args: argparse.Namespace) -> int:
```

Comment:
hmm, we should add the `py.typed` marker to sticht so that it would have actually caught this :p